.github/workflows/changed_file.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+res=$1
+res1=($res)
+echo "${res1} is current location"
+for ii in ${res1[*]}; do
+  cd "examples/${ii}"
+  sh test_ci.sh
+  cd ../../..
```

Comment:
A better way to enter the directory and go back to the original directory after the work is done.

```
pushd "examples/${ii}" > /dev/null
    doSomething
popd < /dev/null
```

Reply:
`ii` is a bad variable name.